examples/atlas-api-key/create-api-key-assign-to-multiple-projects/variables.tf

```
@@ -0,0 +1,14 @@
+variable "public_key" {
+  description = "Public API key to authenticate to Atlas"
+}
+variable "private_key" {
+  description = "Private API key to authenticate to Atlas"
+}
+variable "project_id" {
+  description = "Atlas project name"
+  default     = ""
+}
+variable "additional_project_id" {
+  description = "Atlas project name"
```

Description needs an update
```suggestion
variable "additional_project_id" {
  description = "Atlas project id"
```